src/common/njord.ts

```diff
@@ -531,6 +539,23 @@ const canAward = async ({
   }
 };
 
+export const awardSquad = async (
+  props: AwardInput,
+  ctx: AuthContext,
+): Promise<TransactionCreated> => {
+  const sourceId = props.flags?.sourceId;
+  if (!sourceId) {
+    throw new ForbiddenError('You can not award this Squad');
+  }
+  await ctx.con.manager.getRepository(Source).findOneOrFail({
+    select: ['id'],
+    where: {
+      id: sourceId,
+    },
+  });
```

Comment:
Do we want to check if user is part of this squad? Maybe query source_member?

Also do we want to check if it is a squad, not regulard source?

Reply:
Hmm, not sure it matters a lot for this notification?

Reply:
Yeah but it will cost you cores no?

I mean you don't need to be part of the squad to award it.
We could check that it's a squad for now?